src/index.js

```diff
@@ -52,6 +52,7 @@ const defaultI18n = {
     const formatter = new Intl.NumberFormat(toValidIntlLocale(locale), {
       style: 'currency',
       currency: currencyCode,
+	  minimumFractionDigits: 3,
```

Comment:
This works, but a better approach would be to not change existing behaviour. You'll want to add an optional param to the function for number of digits. The default should leave the current behaviour unchanged ( digits = 2 ).

Reply:
Good catch!